### PR TITLE
Slightly improved the clarity of the parallel docs

### PR DESCRIPTION
**Pull request by BelleNottelling**
Hi all,
Thanks for making Rector, it's proved to be a very handy tool!

This pull request just makes a small change to the "Troubleshooting Parallel" documentation to improve the clarity on how you actually set these options.

### CODE REVIEW

File: resources/docs/troubleshooting-parallel.md [hunk 1]
```diff
@@ -32,6 +31,11 @@ public function parallel(
 ): void
 ```
 
+To manually set these options, call the `parallel` function within the Rector config class:
+```php
+$rectorConfig->parallel(120, 16, 20);
```
Comment (samsonasik) [inline resources/docs/troubleshooting-parallel.md:36]: Rector 0.19.0 is using `60, 32, 20` as default value, ref

https://github.com/rectorphp/rector-src/blob/ac072bc8af186b2c86256aa7d29e52bb608ece27/src/Config/RectorConfig.php#L91

Could you update the default value on line 28 to 30 above it?

Comment (BelleNottelling) [reply resources/docs/troubleshooting-parallel.md:36]: Done :)

**Comment by samsonasik** [comment]
Thank you @BelleNottelling

**Comment by BelleNottelling** [comment]
> Thank you @BelleNottelling

You're welcome!
Thanks again for Rector, it really is a great tool

**Pull request merged**